code/modules/medical/computer/cloning.dm

```diff
@@ -447,12 +447,21 @@
 			break
 	return selected_record
 
+/obj/machinery/computer/cloning/process()
+	update_icon()
```

Comment:
Why would you call this every process, that's incredibly expensive.

Call it when the machine's power state changes, and when it links to other machines.